velox/connectors/hive/storage_adapters/s3fs/tests/S3FileSystemMetricsTest.cpp

```diff
@@ -168,6 +168,34 @@ TEST_F(S3FileSystemMetricsTest, metrics) {
   EXPECT_EQ(1, s3Reporter->counterMap[std::string{kMetricS3GetObjectCalls}]);
 }
 
+// This test takes two to three minutes to run.
```

Comment:
Where does the time go? We should aim to reduce this.

Reply:
I see the delay is from the time out `Couldn't connect to server`. Can we reduce this timeout?

Reply:
I have tried reducing socket and connect timeout values, but still the runtime is the same.